include/linker/common-rom.ld

```diff
@@ -17,6 +17,21 @@
 		__init_end = .;
 	} GROUP_LINK_IN(ROMABLE_REGION)
 
+	SECTION_PROLOGUE(devices,, ALIGN_WITH_INPUT)
```

Comment:
I think that ALIGN_WITH_INPUT shouldn't be necessary for a ROM section, we don't use it on any of the others

Reply:
indeed, that's a left over on my attempts to fix the problem